vimiv/api/__init__.py

```diff
@@ -93,3 +93,19 @@ def add_external_format(
     """
     files.add_image_format(file_format, test_func)
     imagereader.external_handler[file_format] = load_func
+
```

Comment:
Sorry for being a bit of a nuisance here, but personally I would not expose a command that simply wraps the builtin print function via the api. Instead this could just go into `api._modules` as is :blush:

Reply:
I thought that if we already have a *generalized* printer, everyone should also use it. But I also do not mind changing it.

Fixed in b039b82bcb453c33673e671b3d9d2ef7c306f7fa

Reply:
Thanks a lot, this looks great now :blush: